plugin.video.nhlgcl/resources/settings.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Please consider to update to the new settings format at some point

Reply:
Ok, I will look into updating them.